Dockerfile

```diff
@@ -22,13 +21,18 @@ RUN apk --update add \
     rm /var/cache/apk/* && \
     ln -s /usr/bin/php7 /usr/bin/php
 
+COPY composer.json /usr/src/app/composer.json
+COPY composer.lock /usr/src/app/composer.lock
+
 RUN apk --update add curl && \
     curl -sS https://getcomposer.org/installer | php && \
     ./composer.phar install && \
     apk del curl && \
     rm /usr/src/app/composer.phar \
        /var/cache/apk/*
 
+COPY bin/build-content /usr/src/app/bin/build-content
```

Comment:
The behavior is a little confusing. Looks like when the `<dest>` has a trailing slash, the contents of the `<src>`s are copied in. If you want to copy an entire directory, you need to specify a specific destination: `COPY bin ./bin`

Unfortunately this means you can't mix and match so I think keeping the `bin/build-content` COPY line where you have it is fine 👌